feature-cancel-insurance/src/main/kotlin/com/hedvig/android/feature/cancelinsurance/navigation/CancelInsuranceDestinations.kt

```diff
@@ -0,0 +1,68 @@
+package com.hedvig.android.feature.cancelinsurance.navigation
+
+import androidx.compose.material3.windowsizeclass.WindowSizeClass
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavHostController
+import androidx.navigation.navOptions
+import com.hedvig.android.feature.cancelinsurance.CancelInsuranceViewModel
+import com.hedvig.android.feature.cancelinsurance.InsuranceId
+import com.hedvig.android.feature.cancelinsurance.ui.result.CancellationSuccessDestination
+import com.hedvig.android.feature.cancelinsurance.ui.terminationdate.TerminationDateDestination
+import com.kiwi.navigationcompose.typed.Destination
+import com.kiwi.navigationcompose.typed.composable
+import com.kiwi.navigationcompose.typed.createRoutePattern
+import com.kiwi.navigationcompose.typed.navigate
+import com.kiwi.navigationcompose.typed.navigation
+import com.kiwi.navigationcompose.typed.popUpTo
+import kotlinx.serialization.Serializable
+import org.koin.androidx.compose.koinViewModel
+import org.koin.core.parameter.parametersOf
+
+internal sealed interface Destinations : Destination {
+  @Serializable
+  object CancelInsurance : Destinations
+}
+
+internal sealed interface CancelInsuranceDestinations : Destination {
+  @Serializable
+  object TerminationDate : CancelInsuranceDestinations
+
+  @Serializable
+  object TerminationSuccess : CancelInsuranceDestinations
+}
```

Comment:
And these are the possible screens *inside* this feature. In our case this is now picking the date, and the success screen. Here would fit the error screen, the confirmation screen, and so on, from Figma.

p.s. I wanna rename everything from "cancelinsurance to insurance-termination, I've made a bit of a mess using both terms interchangeably